fbw-a32nx/src/systems/fmgc/src/flightplanning/new/FlightPlanService.ts

```diff
@@ -467,4 +467,44 @@ export class FlightPlanService implements FlightPlanInterface {
 
         this.flightPlanManager.copy(FlightPlanIndex.Active, FlightPlanIndex.Temporary, CopyOptions.IncludeFixInfos);
     }
+
+    setCruiseFlightLevel(fl: number, planIndex = FlightPlanIndex.Active) {
+        this.flightPlanManager.get(planIndex).setCruiseFlightLevel(fl);
+    }
+
+    getCruiseFlightLevel(planIndex = FlightPlanIndex.Active) {
```

Comment:
What is the reasoning for introducing getter methods? We get everything else directly from the flight plan objects - this class has dedicated modification methods because those things need to be able to happen over RPC - obtaining data in the plan can happen locally, as we have copies of them.